### PR TITLE
fix to render Stringable content in Accordion item

### PR DESCRIPTION
**Pull request by albertborsos**
| Q             | A
| ------------- | ---
| Is bugfix?    | ✔️
| New feature?  | ❌
| Breaks BC?    | ❌
| Fixed issues  | https://github.com/yiisoft/yii-bootstrap5/commit/e656fe39620102ad90ab33194dd29136e912808f#r70189673

### CODE REVIEW

**Comment by vjik** [comment]
@albertborsos thank you for PR!

Can you add test for this change, please?

**Comment by albertborsos** [comment]
@vjik I made it, but there somthing to discuss.

In my test cases an `.accordion-item` has only one `.accordion-body`. But in the previous tests there is multiple `.accordion-body` in an `.accordion-item`.

I think the previous tests are uses a wrong accordion syntax.

**Comment by albertborsos** [comment]
FYI: on php7.4 a `\Yiisoft\Html\Tag\Base\Tag` instance is not a `Stringable` object, even if `symfony/polyfill-php80` is installed.

**Comment by vjik** [comment]
@albertborsos We can check exists of method `__toString()` instead of `is_a()`. This is resolve problem with stringable objects.

> In my test cases an .accordion-item has only one .accordion-body. But in the previous tests there is multiple .accordion-body in an .accordion-item.
> I think the previous tests are uses a wrong accordion syntax.

@terabytesoftw What you think about this? Item can have several body?

File: src/Accordion.php [hunk 1]
```diff
@@ -478,18 +476,28 @@ private function renderCollapse(array $item, int $index): string
...
+            $items .= (string) $value;
         }
 
-        return $items;
+        return Html::div($items, ['class' => 'accordion-body'])
+            ->encode($this->encodeTags)
+            ->render();
+    }
+
+    private function isAStringableObject($value): bool
```
Comment (samdark) [inline src/Accordion.php:499]:
```suggestion
    private function isStringableObject($value): bool
```

File: src/Accordion.php [hunk 2]
```diff
@@ -478,18 +476,28 @@ private function renderCollapse(array $item, int $index): string
...
+
+        if ($this->isAStringableObject($item['content'])) {
+            $content = [$item['content']];
+        } else {
+            $content = (array) $item['content'];
+        }
 
         foreach ($content as $value) {
-            if (!is_string($value) && !is_numeric($value) && !is_a($value, Stringable::class)) {
+            if (!is_string($value) && !is_numeric($value) && !$this->isAStringableObject($value)) {
```
Comment (samdark) [inline src/Accordion.php:487]:
```suggestion
            if (!is_string($value) && !is_numeric($value) && !$this->isStringableObject($value)) {
```

**Review by samdark — APPROVED** [review]

**Review by terabytesoftw — APPROVED** [review]

**Pull request merged**